plugins/vscode/.vscode/launch.json

```diff
@@ -7,7 +7,7 @@
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
+            "args": ["--extensionDevelopmentPath=${workspaceRoot}", "${workspaceRoot}/../../test/foo", "${workspaceRoot}/../../test/foo/Root.hs" ],
```

Comment:
Is this deliberate?

Reply:
Yes, the `../../test/foo` directory within this repo is very convenient for developing/debugging. I used it all the time when working on this PR. IMO it's a sensible default that can be changed pretty easily.